documentation/src/test/java/example/RepeatedTestsDemo.java

```diff
@@ -50,13 +50,13 @@ void repeatedTestWithRepetitionInfo(RepetitionInfo repetitionInfo) {
 	@RepeatedTest(value = 1, name = "{displayName} {currentRepetition}/{totalRepetitions}")
 	@DisplayName("Repeat!")
 	void customDisplayName(TestInfo testInfo) {
-		assertEquals(testInfo.getDisplayName(), "Repeat! 1/1");
+		assertEquals("Repeat! 1/1", testInfo.getDisplayName());
```

Comment:
Please go ahead and push this to `master` since it is not related to this PR and should be adopted ASAP anyway.

Reply:
Done via https://github.com/junit-team/junit5/commit/d024097eb5e6b5ffd28edfc783ecf7a2512dd27e